tasties_app/tests/conftest.py

```diff
@@ -183,3 +183,41 @@ def recipe():
 def signed_up_credentials():
     user = User.objects.create_user(username=VALID_USER, email=VALID_EMAIL, password=VALID_PASSWORD)
     user.save()
+
+
+@pytest.fixture
+def recipes_filtered_by_salad():
```

Comment:
This looks to me like a very curious name, I'm afraid that it's not descriptive of what this fixture does.

Reply:
i change.